### PR TITLE
UI components: Loading-Spinner, Spacer-Component, Button-Update, TabContentWrapper

### PR DESCRIPTION
**Pull request by niklasdfx**

### CODE REVIEW

**Review by Krysh90 — APPROVED** [review]

File: src/stories/StyledSpacer.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+export interface StyledSpacerProps {
+  showLine?: boolean;
+  spacing: number;
+}
+
+export default function StyledSpacer({ showLine, spacing }: StyledSpacerProps) {
+  let spacerClasses = '';
+
+  showLine ? (spacerClasses = ' border-t border-dfxGray-400') : null;
```
Comment (davidleomay) [inline src/stories/StyledSpacer.tsx:9]:
```ts
const spacerClasses = showLine ? 'border-t border-dfxGray-400' : '';
```

File: src/stories/StyledTabContentWrapper.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { PropsWithChildren } from 'react';
+import { IconVariant } from './DfxIcon';
+import StyledIconButton from './StyledIconButton';
+
+export interface StyledTabContentWrapperProps extends PropsWithChildren {
+  showBackArrow?: boolean;
+  onBackClick: () => void;
```
Comment (davidleomay) [inline src/stories/StyledTabContentWrapper.tsx:7]: `onBackClick` should not be required if back arrow is not required.

File: src/stories/StyledSpacer.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+  const margin = 4 * spacing;
+
+  return <div style={{ margin: margin + 'px 0' }} className={spacerClasses}></div>;
```
Comment (davidleomay) [inline src/stories/StyledSpacer.tsx:13]: Can use string interpolation, if it makes sense for you
```ts
const margin = `${4 * spacing}px 0`;
```

Comment (niklasdfx) [reply src/stories/StyledSpacer.tsx:13]: well, i need this calculation from within different components, now it is in a sepearte function in LayoutFunctions.tsx ... so i guess i leave it like that.

File: src/stories/StyledTabContentWrapper.tsx (hunk 2 above)
Comment (niklasdfx) [reply src/stories/StyledTabContentWrapper.tsx:7]: you're right, thanks.

**Pull request merged**